javascript/javascript3/week2/Currency_Converter/index.html

```diff
@@ -0,0 +1,409 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script src="script.js" defer></script>
+    <title>Currency calculator</title>
+  </head>
+  <body>
+    <main>
+      <h1>Currency Calculator</h1>
+      <div class="main-div">
+        <div class="result"></div>
+
+        <div class="up-div">
+          <input type="number" class="input-up" aria-label="from convert" />
+          <div class="currency-up">
+            <div class="seperater"></div>
+            <label class="up-span"
+              >Select a Currency
+              <select class="list-up">
+                <option value=""></option>
+                <option value="USD" label="US dollar">USD</option>
+                <option value="EUR" label="Euro">EUR</option>
```

Comment:
From the task: `User can choose a currency to convert from(default should be EUR)`. Therefore, this should be selected by default